### PR TITLE
add env variable, bump nypr-django-for-ember

### PR DESCRIPTION
**Pull request by bryn19**

### CODE REVIEW

File: .env.sample [hunk 1]
```diff
@@ -1,4 +1,5 @@
 WQXR_URL=https://wqxr.demo2.wnyc.net
+WQXR_LEGACY=https://http://wqxr-legacy.demo.nypr.digital
```
Comment (ncastro-nypr) [inline .env.sample]: ?

File: config/environment.js [hunk 2]
```diff
@@ -57,7 +57,8 @@ module.exports = function(environment) {
     showsAPIKey: "symphony",
     moreShowsDiscoverStation: "archived-shows",
     moreShowsAPIKey: "mammoth",
-    webRoot: process.env.WQXR_URL,
+    webRoot: process.env.WQXR_LEGACY,
+    wQXRLegacy: process.env.WQXR_URL,
```
Comment (ncastro-nypr) [inline config/environment.js]: these two are flipped?

File: config/environment.js [hunk 3]
```diff
@@ -135,6 +136,7 @@ module.exports = function(environment) {
     ENV.adminRoot = 'http://admin.example.com';
     ENV.etagAPI = 'http://example.com/api/v1/browser_id/';
     ENV.webRoot = 'http://example.com';
+    ENV.wQXRLegacy = 'http://example.com';
```
Comment (ncastro-nypr) [inline config/environment.js]: needs a capital W?

Comment (bryn19) [reply config/environment.js]: this is right you fool

Comment (bryn19) [reply config/environment.js]: also happy birthday!

**Pull request merged**